server/aesdsocket.c

```diff
@@ -178,10 +216,282 @@ int socket_connect(int socketfd, char* ip_addr)
        	syslog(LOG_ERR, "ERROR: Unable to get the IP address");	
        }
        syslog(LOG_INFO, "Accepted connection from %s", ip_addr);
+       ip_address = ip_addr;
```

Comment:
@charan1811 what does it mean to use a global value here to reference in a thread when the value is set outside the thread?  What could happen if multiple connections were handled at once?